app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -278,8 +281,25 @@ class TopicRevisionFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicRevisionFragment_createFragmentWithProfileId_verifyProfileIdInBundle() {
+    launchTopicActivityIntent(
+      profileId,
+      topicId = FRACTIONS_TOPIC_ID
+    ).use {
+      it.onActivity {
+        activity ->
```

Comment:
```suggestion
      it.onActivity { activity ->
```

Wrapping is a bit off here.

Reply:
Updated.